### ISSUE TITLE
Survey Task - styling issues: incomplete image & button display, nav button visibility

### ISSUE THREAD

**Issue by lcjohnso** [zooniverse/front-end-monorepo#4557]
## Describe the bug

1. Choice descriptions are missing in FEM for Gravity Spy ([link](https://frontend.preview.zooniverse.org/projects/zooniverse/gravity-spy/classify/workflow/1610) and see screenshot below) but do appear for Snapshot Serengeti ([link](https://frontend.preview.zooniverse.org/projects/zooniverse/snapshot-serengeti/classify/workflow/4655)).  
![Screen Shot 2023-05-02 at 3 35 44 PM](https://user-images.githubusercontent.com/2006101/235780992-a3d62086-b88e-415a-a659-936c3ff67961.png)

2. Images and buttons for "Confused With" modal are only partially displayed -- images and buttons appear cutoff at the bottom. See Snapshot Serengeti ([link](https://frontend.preview.zooniverse.org/projects/zooniverse/snapshot-serengeti/classify/workflow/4655) and see screenshot below). I expect buttons and images should be completely visible. Please note: behavior is inconsistent -- sometimes the modal items appear normally.
<img width="1209" alt="Screen Shot 2023-05-01 at 2 01 05 PM" src="https://user-images.githubusercontent.com/2006101/235966404-4b6f8863-40a7-4c93-998f-8fa1573ab1f5.png">

3. Navigation Arrows used to look through example images for species (for carousel that appears at top of species detail once the choice is clicked in the main interface) are very subtle / not readily visible -- see screenshot below where arrow is highlighted by red box. Can the contrast of these be improved, and made visible on either a dark or light background?
![Screen Shot 2023-05-03 at 10 42 01 AM](https://user-images.githubusercontent.com/2006101/235967703-cb883656-308c-4214-afb9-f28681f51aa1.png)

attn: @mcbouslog @seanmiller26

**Comment by eatyourgreens** [comment]
Here’s ‘Confused with’ on my phone. I think I triggered this by scrolling the window. 

![A very short Confused With popup. Both the top and bottom of the dialog are cut off, so that only the description text is visible.](https://user-images.githubusercontent.com/59547/236047339-4f5824ef-7adc-41ba-954a-f54247f317f9.jpeg)

**Comment by eatyourgreens** [comment]
The missing descriptions for Gravity Spy are missing from the workflow translations, which were last updated in 2019.

https://www.zooniverse.org/api/translations?language=en&translated_id=1610&translated_type=workflow
<img width="552" alt="Translation strings for Gravity Spy level 1: Neutron Star Mountain." src="https://user-images.githubusercontent.com/59547/236236491-9a12fbf9-a1b2-4982-ad0c-7f5255737373.png">

**Comment by eatyourgreens** [comment]
Images should be centred, now that #4561 has merged. This one, from the Snapshot Serengeti tutorial, is pretty much illegible though.
![A 200px thumbnail of an example screenshot from the Snapshot Serengeti tutorial. The text is very pixelated, to the point of being illegible.](https://thumbnails.zooniverse.org/x200/panoptes-uploads.zooniverse.org/tutorial_attached_image/17a757c6-6973-48ed-b2cc-93c4b9aefa8b.jpeg)

**Comment by lcjohnso** [comment]
Noting that missing species description issue (item 1) is resolved. Jim was correct in his [comment above](https://github.com/zooniverse/front-end-monorepo/issues/4557#issuecomment-1534876334): the text strings needed to be registered with the translations, probably due to these very old workflows pre-dating the current translations setup, so a small edit + autosave triggered the strings to be saved and populated correctly.

**Comment by mcbouslog** [comment]
Per [the Grommet Carousel stories](https://storybook.grommet.io/?path=/story/media-carousel-custom-themed-custom-controls--custom-carousel&globals=theme:base) changing the carousel controls styling looks straightforward. @seanmiller26 - do you have a recommendation (maybe teal or a shade thereof will work)?

**Comment by seanmiller26** [comment]
@mcbouslog Yes, let's go with the main teal #00979D as it can be seen over dark and light backgrounds.

**Comment by mcbouslog** [comment]
@seanmiller26 - I showed @lcjohnso the teal carousel arrows and we're thinking maybe even bigger or more noticeable somehow? I can increase size or current arrows, or per https://icons.grommet.io/?s=arrow maybe the Carets, or something else, I'm not sure, let us know what you think!

**Comment by seanmiller26** [comment]
Are we currently using Grommet's Previous and Next icons? Is it possible to make them thicker? ie increase the stroke or weight?

Otherwise we could try a larger size and see what that looks like.

**Comment by mcbouslog** [comment]
We're using the Grommet Carousel, which defaults to the Grommet Icons Next and Previous. Any other Grommet Icons will be easy to plug in with a custom theme. Grommet icons support a `size` and `color` property. I'll share some screenshots at different sizes later today, let me know if you'd like me to share any screenshots with different colors or icons.

**Comment by seanmiller26** [comment]
I think sticking with Previous and Next for consistency is the right way to go. But I'd love to see the dark teal #005D69 and gold #F0B200 as options.

**Comment by mcbouslog** [comment]
Here's some screen shots with the icon size "large" and color
- neutral-1 / dark teal:
![dark_teal](https://github.com/zooniverse/front-end-monorepo/assets/12118751/7adf0651-c1af-4c79-86c7-818256a1b4c6)
- neutral-2 / gold:
![gold](https://github.com/zooniverse/front-end-monorepo/assets/12118751/9102ea40-dda3-487e-8c8e-f0741ad34694)
- accent-1 / light teal:
![light_teal](https://github.com/zooniverse/front-end-monorepo/assets/12118751/9572921d-fb12-48f6-af0d-5d9893ba52eb)

**Comment by mcbouslog** [comment]
@seanmiller26 - here are a few images with dirt or sand in the background, as close as I could find to desert in the survey sample images (let me know if there's a particular image you have in mind and want me to check with):
![Screenshot 2023-06-26 at 1 14 17 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/24c11f95-052b-4d44-be62-0d6bde07d006)
![Screenshot 2023-06-26 at 1 14 40 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/1624b244-c8ea-4378-a395-d32bcb2cb90a)
![Screenshot 2023-06-26 at 1 15 02 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/16409896-7509-4fc8-a1a6-341d59579e98)
![Screenshot 2023-06-26 at 1 15 42 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/0ee0ff85-aa6c-4352-8754-143c36c11f32)
![Screenshot 2023-06-26 at 1 16 06 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/4d194989-5b74-4a6f-b428-ce81297f4390)
![Screenshot 2023-06-26 at 1 16 21 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/744470a6-7b47-4b9f-b051-de68e8a8f873)
![Screenshot 2023-06-26 at 1 16 43 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/04735e1f-116f-43f8-bbad-a8ab19ace633)
![Screenshot 2023-06-26 at 1 17 25 PM](https://github.com/zooniverse/front-end-monorepo/assets/12118751/dc4c72f4-ad06-43cd-9193-bcd05efdcac8)

**Comment by seanmiller26** [comment]
Thanks Mark, very helpful to see other terrains. I believe the Gold stands out the best with either a grass or dirt/desert background, however it could be more accessible. I've mocked-up a very minimal fill to make the icons even more visible. Keeping in mind that we don't want to block any parts of the image, the opacity is set to just 20%. Let me know your thoughts.

<img width="954" alt="Screen Shot 2023-06-27 at 11 03 00 AM" src="https://github.com/zooniverse/front-end-monorepo/assets/89043489/55bbc2ea-1159-47f2-be8a-339438a375b1">

<img width="793" alt="Screen Shot 2023-06-27 at 10 59 37 AM" src="https://github.com/zooniverse/front-end-monorepo/assets/89043489/98a290ff-d1a2-48dc-add5-2b632aa1d565">

**Comment by mcbouslog** [comment]
@seanmiller26 - unfortunately the Grommet Carousel buttons have a height that is the full height of the image. I think a user would expect the opaque box around the icons to be the button (when hovering, clicking, or focusing), but the button would be the full height of the image, so it might be a little confusing? If possible, for simplicity and consistency, maybe we stick within the provided Grommet Carousel and Icon props for now? Can revisit if there's similar feedback.

Your recent screenshots include thin lines at the bottom to denote which image is active. I don't think that's been included in previous designs? I think other Carousels (i.e. Tutorial) use pill/radial buttons. Let me know if you'd like me add either (thin lines, pills/radials) or keep as is (without selectors).

**Comment by seanmiller26** [comment]
@mcbouslog I just reviewed the Grommet default behavior and you're right, it already does this but with the full height. Are we able to change that grey/white on-hover color to a darker/black overlay? If not I would approve with the default color and our change of the larger icon / Gold color styling.

Ignore the active image selectors! I was just playing around with some ideas. I don't think it'll be necessary to include in this issue.

**Comment by mcbouslog** [comment]
Unfortunately the on-hover styling isn't editable via the provided theming options - https://v2.grommet.io/carousel#theme.

If needed we can wrap Grommet components with styled-components and force the discussed stylings, but styled-components adds a new layer of complexity (and performance issues). 

I think the gold pops really nicely, especially with the larger arrows, and is good for now, and if need be we can try and change more on a different iteration.
